retrofit/src/main/java/retrofit/RestAdapter.java

```diff
@@ -415,8 +415,8 @@ public Builder setConverter(Converter converter) {
     }
 
     /**
-     * The error handler allows you to customize the type of exception thrown for errors on
-     * synchronous requests.
+     * The error handler allows you to customize
+     * {@link retrofit.client.Response response} exceptions.
```

Comment:
I don't think this is accurate either. I'm pretty sure we now always apply the error handler, regardless of whether it's synchronous, async, or Rx.